samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -63,7 +63,8 @@ static void stream_sent_cb(struct bt_audio_stream *stream)
 	}
 
 	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
-	net_buf_add_mem(buf, mock_data, preset_16_2_1.qos.sdu);
+	net_buf_add_mem(buf, mock_data,
+			preset_16_2_1.qos.sdu - BT_HCI_ISO_DATA_HDR_SIZE);
```

Comment:
I think this is wrong. (Of cause this app is just sending mock data, but it sends the wrong message to people using the sample to build an application.
QoS sdu size is the negotiated over-the-air max SDU size - which do not carry any HCI headers - hence there should be no need to reserve space for the header in the payload area of buf. That would mean you would need to setup the ISO link to be able to carry 128 octets to fit a 120 byte audio frame - hence wasting 8 bytes of air-time for each frame.
The buffers in the host needs to be allocated large enough to hold both the ISO max SDU as well as the HCI header, but the data path should not care about the HCI header when building the SDU.

Reply:
Will update this PR to properly fix the issue. The PSN and TS should not be considered part of the SDU